lib/graphql/language/parser.rb

```diff
@@ -1,6 +1,6 @@
 #
 # DO NOT MODIFY!!!!
-# This file is automatically generated by Racc 1.4.15
```

Comment:
The rake task pinning might be outdated. Happy to change it to 1.4.15 everywhere (including docs).

Reply:
Oops, yeah, I'm assuming that newer is better, let's go for it!